spec/cancan/ability_spec.rb

```diff
@@ -376,6 +376,7 @@ class Container < Hash; end
       expect(e.message).to eq("Access denied!")
       expect(e.action).to eq(:read)
       expect(e.subject).to eq(:foo)
+      expect(e.extra_args.first).to eq(1)
```

Comment:
please add some more tests: check what is the content of `extra_args` in different cases (also when there is no extra_args)